### DISCUSSION TITLE
Is is possible to show opacity to background of the popup, Same like Android in iOS

### DISCUSSION THREAD

**Discussion by Rathish100** [xamarin/XamarinCommunityToolkit#1902] [Q&A]
I'm integrated XCT Popup for my app for both Android and iOS. The only difference I see is there is no dark grey transparent opacity background for iOS like in Android. Do we have any properties to enable that or is there any possible way to achieve it.

@jfversluis 

<img width="388" alt="Screen Shot 2022-09-01 at 12 01 07 PM" src="https://user-images.githubusercontent.com/4198409/187971231-f1ce4b56-2a96-4c30-8d7c-54c310e17207.png">
<img width="378" alt="Screen Shot 2022-09-01 at 12 01 20 PM" src="https://user-images.githubusercontent.com/4198409/187971267-d3cf7cb6-5de0-46a9-a7da-8a388b5a136c.png">

**Comment by pictos** [comment]
@Rathish100, this is odd... Can you share the repo link of this sample?

**Reply by Rathish100** [reply]
I just tried with this sample code
[https://github.com/SkyeHoefling/XCT-Popups-Samples](https://github.com/SkyeHoefling/XCT-Popups-Samples)
